### PR TITLE
many improvements, fix docs

### PR DESCRIPTION
**Pull request by ignaigna**
title, some of the improvements:

- fix README.md example
- follow proper flake8 rules
- cleanup code

### CODE REVIEW

File: rblxopencloud/creator.py
Comment (ignaigna) [reply rblxopencloud/creator.py:267]: black formatter does it like that, maybe doing a custom config would fix this properly?

File: rblxopencloud/creator.py
Comment (treeben77) [inline rblxopencloud/creator.py:467]: >79 character

Comment (ignaigna) [reply rblxopencloud/creator.py:467]: > black formatter does it like that, maybe doing a custom config would fix this properly?

**Review by treeben77 — APPROVED** [review]

**Comment by LawMixer** [comment]
treeben7 forgot to merge my pull request in, so sorry for any of the merge conflicts!

**Comment by treeben77** [comment]
ill let @LawMixer review if they want to before merging

**Pull request merged**
